express/parsers/apps/espresso/formats/espresso_640xml.py

```diff
@@ -98,18 +103,3 @@ def final_basis(self) -> Dict[str, Union[str, Dict]]:
             result["coordinates"].append({"id": atom_id, "value": coords})
 
         return result
-
-    @staticmethod
-    def string_to_vec(string: str, dtype: type = float, sep: Optional[str] = None) -> List[Any]:
-        """
-        Given a string and some delimiter, will create a vector with the specified type.
-
-        Args:
-            string (str): The string to convert, for example "6.022e23 2.718 3.14159"
-            dtype (type): The type to convert into. Must support conversion from a string. Defaults to `float`
-            sep (Optional[str]): Delimiter for the the string. Defaults to whitespace.
-        Returns:
-            List[Any]: A list that has the correct type, for example [6.022e23, 2.718, 3.14159]
-        """
-        result = [dtype(component) for component in string.split(sep)]
-        return result
```

Comment:
Added the ability to choose the container this method produces. Also realized that it's so general, that it can just go into the parent class, for use if we ever need to parse a vector stored in one line in XML.

Reply:
Not sure this is worth an actual function definition at all as it is just a level of indirection around an otherwise succinct list-comprehension, especially given that the `container` type makes assumptions about its constructor. Not suggesting it needs to be removed but to me `np.array([float(s) for s in string.split()])` is more readable than `string_to_vec(string, type=(was it type or dtype I forgot), sep=(the actual argument to split anyway))` as there is no need to understand the "foreign" interface.